runtime/amd64.S

```diff
@@ -315,14 +315,14 @@ FUNCTION(G(caml_call_gc))
         CFI_STARTPROC
 LBL(caml_call_gc):
     /* Record lowest stack address and return address. */
-        movq    (%rsp), %rax
-        movq    %rax, Caml_state(last_return_address)
-        leaq    8(%rsp), %rax
-        movq    %rax, Caml_state(bottom_of_stack)
+        movq    (%rsp), %r11
```

Comment:
Is it really safe to use `%r11` here?  If `X86_proc.use_plt` is false, `destroyed_by_plt_stub` is empty and `destroyed_at_alloc` will be empty too.

Reply:
You're right. (This is what caused the msvc build failure). Just pushed a fix.